<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLineEdit" name="YLine"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="OYLabel">
     <property name="text">
      <string>OY</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QDoubleSpinBox" name="edges_green">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLabel" name="Label_edge_width_2">
     <property name="text">
      <string>Vertices size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="startRecordBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Начать запись</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="OXLine"/>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="Label_edge_red_4">
     <property name="text">
      <string>Red</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="OXLabel">
     <property name="text">
      <string>OX</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_name">
         <property name="text">
          <string>File.obj</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Количество вершин:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="vertexes">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Количество полигонов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="facets">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="addFrameBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Добавить кадр</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2" colspan="2">
    <widget class="QComboBox" name="comboBox_vertices_type">
     <property name="currentText">
      <string>Ничего</string>
     </property>
     <item>
      <property name="text">
       <string>Ничего</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Круг</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Квадрат</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="QPushButton" name="refresh_background_2">
     <property name="text">
      <string>Refresh edges color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="fileBtn">
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QPushButton" name="refresh_background">
     <property name="text">
      <string>Refresh background color</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="Label_bg_blue">
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="XLabel">
     <property name="maximumSize">
      <size>
       <width>10777153</width>
       <height>16777081</height>
      </size>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLabel" name="ScaleLabel_3">
     <property name="text">
      <string>Отображается</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QComboBox" name="comboBox_projection">
     <property name="currentText">
      <string>Параллельная</string>
     </property>
     <item>
      <property name="text">
       <string>Параллельная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Центральная</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="ScaleLabel">
     <property name="text">
      <string>Scale</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QDoubleSpinBox" name="vertices_red">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="OYLine"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="OZLabel">
     <property name="text">
      <string>OZ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="Label_edge_blue">
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="OZLine"/>
   </item>
   <item row="17" column="3">
    <widget class="QPushButton" name="refresh_background_3">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="rotateBtn">
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="background_red">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="shiftBtn">
     <property name="text">
      <string>Shifts</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="Label_edge_width">
     <property name="text">
      <string>Edges width</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="Label_edge_red">
     <property name="text">
      <string>Red</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="ScaleLabel_2">
     <property name="text">
      <string>Проекция</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="21">
    <widget class="OGLWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="Label_bg_red">
     <property name="text">
      <string>Red</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="YLabel">
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QLabel" name="Label_edge_red_3">
     <property name="text">
      <string>Green</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QDoubleSpinBox" name="background_blue">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="21" column="1" colspan="3">
    <widget class="QPushButton" name="refresh_background_4">
     <property name="text">
      <string>Refresh vertices color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="XLine"/>
   </item>
   <item row="19" column="3">
    <widget class="QLabel" name="Label_edge_red_2">
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="ZLine"/>
   </item>
   <item row="18" column="2" colspan="2">
    <widget class="QComboBox" name="comboBox_edges_type">
     <property name="currentText">
      <string>Сплошной</string>
     </property>
     <item>
      <property name="text">
       <string>Сплошной</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Пунктирный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="22" column="3">
    <widget class="QPushButton" name="refresh_background_5">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="ScaleLabel_4">
     <property name="text">
      <string>Тип ребер</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="Label_bg_green">
     <property name="text">
      <string>Green</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="ZLabel">
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QSlider" name="horizontalSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="pageStep">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>1000</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QDoubleSpinBox" name="vertices_size">
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="20" column="3">
    <widget class="QDoubleSpinBox" name="vertices_blue">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="15" column="3">
    <widget class="QDoubleSpinBox" name="edges_blue">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="stopRecordBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Остановить запись</string>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <widget class="QDoubleSpinBox" name="vertices_green">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QDoubleSpinBox" name="edges_width">
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QDoubleSpinBox" name="background_green">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="Label_edge_green">
     <property name="text">
      <string>Green</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QDoubleSpinBox" name="edges_red">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QWidget</extends>
   <header>oglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
